chunkflow/flow.py

```diff
@@ -276,6 +276,30 @@ def normalize_contrast_contrast_cmd(tasks, name, levels_path, mip, clip_fraction
             task['log']['timer'][name] = time() - start
         yield task
 
+@cli.command('normalize-section-mu')
```

Comment:
It would be better that the command name be the same with the operator name: `normalize-section-shang`

Reply:
This was a pun intended as "mu as in μm/μs/etc" (micro, minimal, doesn't require a pre-computed histogram)